app/serializers/api/child_serializer.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module Api
+  class ChildSerializer < ActiveModel::Serializer
+    type 'people'
+    attributes :id, :first_name, :email, :account_active, :role
+
+    def role
```

Comment:
are you sure this works (as in, that this takes preference over object.role? (I guess it does, but just to be sure))

Reply:
Yes. I also test this in the specs.